httpcore5-h2/src/main/java/org/apache/hc/core5/http2/impl/nio/bootstrap/H2ServerBootstrap.java

```diff
@@ -212,7 +213,6 @@ public final H2ServerBootstrap setIOSessionDecorator(final Decorator<IOSession>
         return this;
     }
 
-
```

Comment:
@CoolTomatos  do admit the use of line separators is not very consistent here, but can we avoid formatting changes nevertheless?